proto/live_update/figma/figma_node.proto

```diff
@@ -0,0 +1,37 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package com.android.designcompose.proto.live_update.figma;
```

Comment:
I'm a little confused here, is this file just for live update or is this related to the NodeQuery that we use a bunch in the Kotlin code?

Reply:
I assumed NodeQuery was part of live update. Is it still used even when we don't have live update enabled?

Reply:
Yea, if you look at the generated kotlin code for a designcompose function, it creates a nodequery of the root frame name. That's how we look for views.

Reply:
Ok, so it's not part of the design definition, and it doesn't look like it's sent across the JNI, right? So it's just a struct that we use in both the kotlin and rust code?

Reply:
Yea it's part of the design definition:
```
pub struct SerializedDesignDoc {
    pub last_modified: String,
    pub views: HashMap<NodeQuery, toolkit_schema::View>,
    ...
}
```

Reply:
Ah, that's the issue, the message for definition is incorrect. 
```
message SerializedDesignDoc {
  string last_modified = 1;
  map<string, definition.view.View> views = 2;
```

Fixing!

Reply:
.... And it's not that simple. I can't use a message as a key for a proto map. I'll file a separate ticket for this.